dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -271,41 +271,53 @@ def test_correlation(self, *mock):
         profile_options = dp.ProfilerOptions()
         profile_options.set({"correlation.is_enabled": True})
 
-        # data with one column, correlation not updated
+        # data with one column
         data = pd.DataFrame([1.0, None, 1.0, None, 5.0])
         profiler = dp.StructuredProfiler(data, options=profile_options)
-        self.assertIsNone(profiler.correlation_matrix)
+        expected_corr_mat = np.array([[np.nan]])
```

Comment:
imo, this case should return either `None` or `np.array([[1.0]])` ?

Reply:
Interesting catch, yes it looks like it should be `1.0`

Reply:
Since the data has non-numeric values, should it be np.nan?

Reply:
However:
```
In [25]: np.corrcoef(pd.DataFrame([1, 2]), rowvar=False)
Out[25]: 1.0

In [26]: np.corrcoef(pd.DataFrame([[1], [2]]), rowvar=False)
Out[26]: 1.0
```

Reply:
actually, this case has None value, then the returning value as nan makes sense. But we need to add the case without nan, e.g. 
```
>>> np.corrcoef(np.array([1, 2, 3, 4, 5]), rowvar=False)
1.0
```